### PR TITLE
Us 41

### LINKED ISSUE #42: User Story 41, Admin visits Merchant Index Page
```
As an admin user
When I visit the merchant's index page at "/merchants"
I see all merchants in the system
Next to each merchant's name I see their city and state
The merchant's name is a link to their Merchant Dashboard at routes such as "/admin/merchants/5"
I see a "disable" button next to any merchants who are not yet disabled
I see an "enable" button next to any merchants whose accounts are disabled
```

DO THESE STORIES FIRST:
- User Story 39, Merchant Index Page

### Checklist:

- [x] base tests are written
- [x] base code is written to pass tests
- [x] edge case ("sad path") tests written (if required)
- [x] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #227

child of #274

depends on #226

### PR DESCRIPTION
**Pull request by WHomer**
What does this PR do? 
Resolves #42

### CODE REVIEW

File: app/views/merchants/index.html.erb [hunk 1]
```diff
@@ -4,13 +4,25 @@
...
+          <% end %>
         </div>
         <ul class="list-group list-group-flush">
           <li class="list-group-item text-center">Location: <%= merchant.city %>, <%= merchant.state %></li>
           <li class="list-group-item text-center">Joined: <%= merchant.created_at.strftime("%B %d, %Y") %></li>
+          <% if current_admin? && merchant.active? %>
+            <li class="list-group-item text-center"><%= link_to "Disable", root_path %></li>
+          <% elsif current_admin? && !merchant.active? %>
+            <li class="list-group-item text-center"><%= link_to "Enable", root_path %></li>
+          <% end %>
```
Comment (BrennanAyers) [inline app/views/merchants/index.html.erb:20]: Cool!

File: app/controllers/merchants_controller.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
 class MerchantsController < ApplicationController
 
   def index
-    @merchants = User.all_active_merchants
+    @merchants = User.all_active_merchants if !current_admin?
+    @merchants = User.where(role: :merchant) if current_admin?
```
Comment (zachlp2016) [inline app/controllers/merchants_controller.rb:5]: I think we will try to move this out of the business logic if we have enough time but it's certainly ok for right now.

File: app/views/merchants/index.html.erb [hunk 3]
```diff
@@ -4,13 +4,25 @@
     <section id="merchant-<%= merchant.id %>">
       <div class="card" style="width: 18rem;">
         <div class="card-header text-center">
-          <%= merchant.name %>
+          <% if current_admin? %>
+            <%=link_to merchant.name, "/admin/merchants/#{merchant.id}" %>
+          <% else %>
+            <%= merchant.name %>
+          <% end %>
```
Comment (zachlp2016) [inline app/views/merchants/index.html.erb:11]: Looks great!

**Review by zachlp2016 — APPROVED** [review]
Great job!
